tools/cold-backup/README.md

```diff
@@ -1,26 +1,32 @@
-## Strimzi backup
+## Cold backup
 
-Bash script for offline (_cold_) backups of Kafka clusters on Kubernetes or OpenShift. 
-The script only supports a local file system, and you need to make sure to have enough free disk space in the target directory.
+Bash script for running cold backups of test or development Kafka clusters deployed on Kubernetes.
```

Comment:
I don't think this makes much sense. 
* Why do you need backup for development and test clusters?
* Why is this not good enough for production (assuming you can live with the outage)?

Reply:
> Why do you need backup for development and test clusters?

Maybe you did a fairly complex setup that you want to backup and restore as needed. In some testing/demo environment, K8s clusters are automatically deleted after some time. Here I'm thinking about support teams that works with multiple customers.

> Why is this not good enough for production (assuming you can live with the outage)?

Well, the backup/restore procedures are pretty slow because they use tar that does not support parallel execution. I'm sure that there is a way to make it run all volumes in parallel, but it would be too complex for a bash script. I would say, if users find this tool useful, we could implement it with Quarkus for example. WDYT?

Reply:
I can understand that it has its limitations for production. But TBH:
* What you say there basically says do not use this in production
* I'm not sold on the idea how useful tool it is for development and testing. Especially if it is _slow_ as you describe it.
* A good practice for demos is to setup things from scratch to show how things work. Not just restore some backup.

So, honestly, if it is not good enough for production ... I have to wonder if we are just wasting our time on this and if it should be cut from the project. Because it seems to have very little value.

Reply:
It has been useful in a very limited set of use cases, so I'm ok with cutting it. Should I open a PR for that?

Reply:
It probably should be discussed on spme community call to see what others think.

Reply:
OK, fine. Thanks.